src/pages/student/Bookmarks.js

```diff
@@ -419,11 +431,13 @@ function Bookmarks({
             )}
           </div>
         </div>
-        <div className="results-content">
-          <GridComponent
-            displayBanner={true}
-            bookmarks={student.bookmarked_clubs}
-          />
+        <div className={student.bookmarked_clubs.length > 9 ? "bookmarks-scroll" : null}>
+          <div className="results-content" style={{marginRight: '10px'}}>
```

Comment:
hi @satiyak just UATed and the mobile view and margin look great! there still seems to be this issue where the GridComponent correctly collapses into two columns for recommended clubs but not the user's bookmarked clubs however. could you please take a look at the screenshot below? when i resize horizontally, the cards in the bookmarked section disappear into the x-overflow

<img width="832" alt="Screen Shot 2021-11-09 at 9 49 30 AM" src="https://user-images.githubusercontent.com/46834662/140977720-d94503c5-a285-4e7c-81e8-807ba4a9c632.png">